Remotion/Development/Web/UnitTesting/UI/Controls/Rendering/HtmlHelperBase.cs

```diff
@@ -294,6 +294,37 @@ public void AssertNoAttribute (XmlNode node, string attributeName)
       AssertIsNull(node.Attributes[attributeName], "Attribute '{0}' is present although it should not be.", attributeName);
     }
 
+    /// <summary>
+    /// Asserts that <paramref name="element"/> has a single descending element with the specified <paramref name="id"/> and returns the node.
+    /// </summary>
+    public XmlNode GetAssertedElementById (XmlNode element, string id)
```

Comment:
Please check, I think we have tests for these type of methods (`HtmlHelperBaseTest`)